packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -476,6 +493,9 @@ export class EntityInsertPanelImpl extends ReactN.Component<Props, StateProps> {
                 };
             },
             () => {
+                if (this.props.onParentChange) {
+                    this.props.onParentChange(updatedModel.entityParents);
+                }
```

Comment:
same here, I think the syntax is 
```suggestion
                this.props?.onParentChange(updatedModel.entityParents);
```